apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/PatientE2EBase.java

```diff
@@ -0,0 +1,765 @@
+package gov.cms.bfd.server.war;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.hasKey;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.not;
+
+import ca.uhn.fhir.model.primitive.DateTimeDt;
+import gov.cms.bfd.model.codebook.data.CcwCodebookVariable;
+import gov.cms.bfd.model.rif.entities.Beneficiary;
+import gov.cms.bfd.model.rif.entities.BeneficiaryHistory;
+import gov.cms.bfd.model.rif.samples.StaticRifResource;
+import gov.cms.bfd.model.rif.samples.StaticRifResourceGroup;
+import gov.cms.bfd.server.war.commons.CCWUtils;
+import gov.cms.bfd.server.war.commons.TransformerConstants;
+import gov.cms.bfd.server.war.r4.providers.R4PatientResourceProvider;
+import io.restassured.response.Response;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Stream;
+import org.hl7.fhir.r4.model.Patient;
+import org.junit.jupiter.api.Test;
+
+/** Abstract base class for RestAssured Patient E2E tests shared between BFD versions. */
+public class PatientE2EBase extends ServerRequiredTest {
+
+  /** The base patient endpoint. */
+  protected static String patientEndpoint;
```

Comment:
Why is this static?  I'd think it could be non-static final and passed to its constructor from derived class constructors.  Two different derived classes setting this static to two different values might create problems in parallel testing scenarios.

Reply:
Good point, had it being set once at some point and changed that, but forgot to remove the static in the base class.